src/containers/BookList.js

```diff
@@ -2,16 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import Book from '../components/Book';
-import { deleteBook } from '../actions';
+import { createBook, deleteBook } from '../actions';
 
-const BookList = ({ books, delete: handleRemoveBook }) => (
-  <div>
-    {
-      books.map(
-        book => <Book book={book} key={book.id} delete={handleRemoveBook} />,
-      )
-    }
-  </div>
+const BookList = ({ books }) => (
+  books.map(
+    book => <tr key={book.id}><Book book={book} /></tr>,
```

Comment:
Please implement the `handleRemoveBook` function and pass it as a prop to the Book component.